scripts/upgrade

```diff
@@ -56,8 +56,16 @@ current_major_version=${current_version%%.*}
 
 if [ $current_major_version -gt 11 ]
 then
+    # Inform the backup/restore process that it should not save the data directory
+    # Use only for the previous backup script that doesn't set 'is_big'
+    ynh_app_setting_set $app backup_core_only 1
```

Comment:
`if [ $current_major_version -lt 15 ]` maybe?

Reply:
This variable will just be ignored if the backup script is recent